ksm/ksm_day7/모의고사.js

```diff
@@ -0,0 +1,26 @@
+function solution(answers) {
+    const answer = [];
+    const supo = [
+        [1,2,3,4,5], 
+        [2,1,2,3,2,4,2,5], 
+        [3,3,1,1,2,2,4,4,5,5] 
+    ];
+    
+    const score = [];
```

Comment:
```suggestion
    const answer = [];
    const patterns = [
        [1, 2, 3, 4, 5], 
        [2, 1, 2, 3, 2, 4, 2, 5], 
        [3, 3, 1, 1, 2, 2, 4, 4, 5, 5] 
    ];
    const score = [];
```
- ㅎㅎㅎ소리나는대로 영어로 표기식도 종종 쓰이긴하지만 현재 코드에서는 `patterns`가 어떨까요? (배열은 복수형이 일반적)

Reply:
배열은 여러 요소를 담는 구조이니 복수형 네이밍이 적절한 것 같습니다. 참고하겠습니다!